pandas/core/arrays/masked.py

```diff
@@ -404,6 +404,8 @@ def round(self, decimals: int = 0, *args, **kwargs):
         DataFrame.round : Round values of a DataFrame.
         Series.round : Round values of a Series.
         """
+        if self.dtype.kind == "b":
+            return self
```

Comment:
We don't need to return a copy here?

Reply:
no the block level handles this properly